### PR TITLE
First implementation of responsive-layout blocks

### PR DESCRIPTION
**Pull request by victorhmp**
#### What does this PR do?

Creates four new blocks:

- `responsive-layout.desktop`
- `responsive-layout.mobile`
- `responsive-layout.tablet`
- `responsive-layout.phone`

Each of those blocks has composition `children`, and will only render their children in the correct screen-sizes.

#### How to test it? \*

Just resize the window and see the magic take place:
https://victorhmp--storecomponents.myvtex.com/about-us

### CODE REVIEW

**Comment by victorhmp** [comment]
@lbebber this is the most straightforward implementation possible, but it might not be the smartest hahahah I don't know if we need something more refined than that

File: react/ResponsiveLayoutDummy.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const ResponsiveLayoutDummy = () => {
+  console.warn(
+    'You are inserting the `responsive-layout` block directly. Please use `responsive-layout.desktop`, `responsive-layout.mobile`, `responsive-layout.tablet` or `responsive-layout.phone` instead.'
+  )
+
+  return <span />
```
Comment (klzns) [inline react/ResponsiveLayoutDummy.tsx]: Return null

File: react/ResponsiveLayoutDummy.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const ResponsiveLayoutDummy = () => {
+  console.warn(
```
Comment (klzns) [inline react/ResponsiveLayoutDummy.tsx]: Console.error

**Review by klzns — CHANGES_REQUESTED** [review]

**Review by klzns — DISMISSED** [review]

File: docs/README.md [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+- `responsive-layout.tablet`
+- `responsive-layout.phone`
+
+Each of them have `composition: children`, which means that they expect to receive an array of `children` blocks to be rendered by them, if the current screen-size is right for their breakpoint.
+
+## Blocks API
+
+These blocks are defined by the interfaces below:
+
+```json
```
Comment (klzns) [inline docs/README.md:18]: I think this part of our docs is useless.

I would jump right into some examples of usage.

**Review by iaronaraujo — APPROVED** [review]

**Pull request merged**